google-cloud-storage/src/test/java/com/google/cloud/storage/V4PostPolicyTest.java

```diff
@@ -166,17 +154,25 @@ public void test() {
             style);
 
     String expectedPolicy = testData.getPolicyOutput().getExpectedDecodedPolicy();
+
     StringBuilder escapedPolicy = new StringBuilder();
 
     // Java automatically unescapes the unicode escapes in the conformance tests, so we need to
     // manually re-escape them
-    for (char c : expectedPolicy.toCharArray()) {
+    char[] expectedPolicyArray = expectedPolicy.toCharArray();
```

Comment:
This appears to be reproducing the algorithm in the test. That risks copying bugs from one place to another, and verifying the buggy behavior. Can you make the expected output a literal?

Reply:
Yeah, they both happen to need to do the same thing (escaping certain characters). Sorry, I'm unsure what you mean by "make it a literal," can you elaborate?